data/i18n/en/en.toml

```
@@ -60,6 +60,9 @@ other = "Older versions"
 [end_of_life]
 other = "End of Life:"
 
+[envvars_heading]
+other = "Environment Variables"
```

nit: The style guide suggests that this should be in sentence case (hard to tell, because the See Also heading is also in the wrong case).
```suggestion
other = "Environment variables"
```